src/components/dialogs/input-hooks.js

```diff
@@ -442,51 +476,131 @@ export const useAutocompleteField = ({
         }
     }, [selectedValue]);
 
+    useEffect(() => {
+        if (!values || typeof values === 'function') {
+            setIsLoading(false);
+            return;
+        }
+        if (typeof values?.then === 'function') {
+            setIsLoading(true);
+        }
+        const valuePromise = Promise.resolve(values);
+        valuePromise.then((vals) => {
+            setPresentedOptions(vals);
+            setIsLoading(false);
+            if (values?.length === 0) setExpanded(false);
+        });
+    }, [values]);
+
+    const handleForcedSearch = useCallback(
+        (term) => {
+            if (!onSearchTermChange) return;
+            setIsLoading(true);
+            setExpanded(true);
+            onSearchTermChange(term, true);
+        },
+        [onSearchTermChange]
+    );
+
+    const myOnOpen = useCallback(() => {
```

Comment:
change function name to something more clear

Reply:
Ja wohl !